### PR TITLE
Add exchanges to README

### PR DESCRIPTION
**Pull request by jhepkema**
Added the exchanges with description + content listing. Added a few exchanges as comments since they are not currently available for buying NEO, but will probably be once the Chinese situation is resolved.

<i>Exchanges are awesome because we need to be able to buy NEO! </i>

### CODE REVIEW

**Comment by revett** [comment]
@jhepkema Thanks for the PR!

File: README.md [hunk 1]
```diff
@@ -67,6 +68,36 @@
...
+*English interface*
+- [HitBTC](https://hitbtc.com/NEO-to-ETH) - A European exchange that allows trading **NEO** for BTC, ETH and USDT.  
+*English interface*
+- [Cryptopia.co](https://www.cryptopia.co.nz/Exchange?market=NEO_BTC) - A New Zealand-based exchange that allows trading **NEO** for BTC, LTC and DOGE.  
+*English interface*
+- [JUBI](https://www.jubi.com/coin/ans) - A Chinese exchange that allows trading **NEO** for Chinese Yuan Renminbi (CNY).  
+*Chinese interface*
+- [Poloniex](https://www.19800.com/trade.html?marketid=cny_gas) - A United States-based exchange that allows trading **GAS** for BTC and ETH.  
+*English interface*
+<!-- - [YoBTC](https://www.yobtc.com/trade/index.shtml?coinId=15) - currently out of service, see https://twitter.com/yobtc_platform/status/908868690187239424 -->
```
Comment (revett) [inline README.md]: Would remove until exchanges support NEO.

File: README.md [hunk 2]
```diff
@@ -67,6 +68,36 @@
...
+*English interface*
+- [Changelly](https://changelly.com/) - A Czech-based exchange that allows you to swap **NEO** with various other cryptocurrencies.  
+*English interface*
+- [Coinnest](https://www.coinnest.co.kr/market-neo) - A Korean exchange that allows trading between **NEO** and South-Korean Won (SKW).  
+*Korean and Chinese interface available*
+- [CoinSpot](https://www.coinspot.com.au/buy/ans) - An Australian exchange that allows trading between **NEO** and Australian Dollars (AUD).  
+*English interface*
+- [CoinSwitch](https://www.coinswitch.co/exchange/btc/neo) - An Indian exchange that allows you to swap **NEO** with various other cryptocurrencies.  
+*English interface*
+- [HitBTC](https://hitbtc.com/NEO-to-ETH) - A European exchange that allows trading **NEO** for BTC, ETH and USDT.
```
Comment (revett) [inline README.md]: A-Z order of links please.

File: README.md [hunk 3]
```diff
@@ -67,6 +68,36 @@
 - [NEPs](https://github.com/neo-project/proposals) - NEPs describe standards for the NEO platform.
 - [Network Protocol](http://docs.neo.org/en-us/node/network-protocol.html) - Explanation of how NEO nodes communicate.
 
+## Exchanges
+
+- [Anycoindirect](https://anycoindirect.eu/en/buy/neo) - A Dutch exchange that accepts iDeal payments for **NEO**.
```
Comment (revett) [inline README.md]: `iDeal` should be `iDEAL`

File: README.md [hunk 4]
```diff
@@ -67,6 +68,36 @@
 - [NEPs](https://github.com/neo-project/proposals) - NEPs describe standards for the NEO platform.
 - [Network Protocol](http://docs.neo.org/en-us/node/network-protocol.html) - Explanation of how NEO nodes communicate.
 
+## Exchanges
+
+- [Anycoindirect](https://anycoindirect.eu/en/buy/neo) - A Dutch exchange that accepts iDeal payments for **NEO**.  
+*English, Dutch, German, French, Spanish and Italian interfaces*
+- [Binance](https://www.binance.com/tradeDetail.html?symbol=NEO_BTC) - A Hong Kong-based exchange that allows trading **NEO** and **GAS** with BTC. Will return GAS when you hold NEO on their exchange.  
+*English, Chinese, Korean, Japanese, Russian and Spanish interfaces*
+- [Bitfinex](https://www.bitfinex.com/stats) - A Hong Kong-based exchange where you can trade **NEO** with BTC.
```
Comment (revett) [inline README.md]: Please see CONTRIBUTING.md about not including `NEO` in the description of links.

**Comment by jhepkema** [comment]
Thanks for the feedback @revett, any more suggestions?

File: README.md [hunk 5]
```diff
@@ -67,6 +68,33 @@
...
+- [Anycoindirect](https://anycoindirect.eu/en/buy/neo) - A Dutch exchange that accepts iDEAL payments.  
+*English, Dutch, German, French, Spanish and Italian interfaces*
+- [Binance](https://www.binance.com/tradeDetail.html?symbol=NEO_BTC) - A Hong Kong-based exchange that allows trading with BTC. Returns GAS to users.  
+*English, Chinese, Korean, Japanese, Russian and Spanish interfaces*
+- [Bitfinex](https://www.bitfinex.com/stats) - A Hong Kong-based exchange where you can trade with BTC.  
+*English, Russian and traditional and simplified Chinese interfaces*
+- [Bittrex](https://bittrex.com/Market/Index?MarketName=BTC-NEO) - A United States-based exchange. Allows trading for BTC, ETH and USDT.  
+*English interface*
+- [Changelly](https://changelly.com/) - A Czech-based exchange that allows trading for various other cryptocurrencies.  
+*English interface*
```
Comment (revett) [inline README.md]: If there is only an `English interface`, then I don't think it needs to be stated.

File: README.md [hunk 6]
```diff
@@ -67,6 +68,33 @@
...
 - [Network Protocol](http://docs.neo.org/en-us/node/network-protocol.html) - Explanation of how NEO nodes communicate.
 
+## Exchanges
+
+- [Anycoindirect](https://anycoindirect.eu/en/buy/neo) - A Dutch exchange that accepts iDEAL payments.  
+*English, Dutch, German, French, Spanish and Italian interfaces*
+- [Binance](https://www.binance.com/tradeDetail.html?symbol=NEO_BTC) - A Hong Kong-based exchange that allows trading with BTC. Returns GAS to users.  
+*English, Chinese, Korean, Japanese, Russian and Spanish interfaces*
+- [Bitfinex](https://www.bitfinex.com/stats) - A Hong Kong-based exchange where you can trade with BTC.  
+*English, Russian and traditional and simplified Chinese interfaces*
```
Comment (revett) [inline README.md]: Put these interface description on the same line as the link.

**Comment by jhepkema** [comment]
Should be better now @revett :)

**Pull request merged**